src/bundle/Resources/public/js/scripts/fieldType/ezcountry.js

```diff
@@ -1,44 +1,51 @@
-(function(global, doc, eZ) {
+(function (global, doc, eZ) {
     const SELECTOR_FIELD = '.ez-field-edit--ezcountry';
-
-    class EzCountryValidator extends eZ.BaseFieldValidator {
+    const SELECTOR_SELECTED = '.ez-custom-dropdown__selection-info';
+    const SELECTOR_SOURCE_INPUT = '.ez-data-source__input';
+    const EVENT_VALUE_CHANGED = 'valueChanged';
```

Comment:
IF it's our event, use sth like ezValueChanged (IbexaValueChanged?) to avoid collisions